keps/sig-auth/0014-dynamic-audit-configuration.md

```diff
@@ -114,27 +114,27 @@ type WebhookBackend struct {
 // but may require some additive auth parameters
 type WebhookClientConfig struct {
     // URL of the server
-    URL *string `json:"url,omitempty"`
+    URL *string
 
     // Service name to send to
-    Service *ServiceReference `json:"service,omitempty"`
+    Service *ServiceReference
 
     // `caBundle` is a PEM encoded CA bundle which will be used to validate
     // the webhook's server certificate.
-    CABundle []byte `json:"caBundle"`
+    CABundle []byte
 }
 ```
 
-This configuration object will be accepted both as the existing runtime flags, and a dynamic api object. Multiple definitions can exist as independent solutions. These updates will require the audit API to be registered with the apiserver. Dynamic configuration will be enabled by a feature gate. If existing flags are provided to configure the audit backend they will be taken as a separate configuration object.
+Multiple definitions can exist as independent solutions. These updates will require the audit API to be registered with the apiserver. The dynamic configurations will be wrapped by truncate and batch options, which are set statically through existing flags. Dynamic configuration will be enabled by a feature gate for pre-stable releases, as well as a runtime flag `--audit-dynamic-enabled`. If existing flags are provided to configure the audit backend they will be taken as a separate backend configuration.
```

Comment:
I think the feature gate is sufficient... I don't think we also need the `--audit-dynamic-enabled` flag.